yt_dlp/networking/_websockets.py

```diff
@@ -53,6 +69,22 @@
     websockets.sync.connection.Connection.recv_events_exc = None
 
 
+class WebsocketsLoggingHandler(logging.Handler):
+    """Redirect websocket logs to our logger"""
+
+    def __init__(self, logger, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._logger = logger
+
+    def emit(self, record):
+        try:
+            msg = self.format(record)
+            self._logger.stdout(msg)
+
+        except Exception:
+            self.handleError(record)
```

Comment:
```suggestion
    def emit(self, record):
        try:
            msg = self.format(record)
        except Exception:
            self.handleError(record)
        else:
            self._logger.stdout(msg)
```
So any logging errors don't get captured

Reply:
looks like this caught a problem in the test suite where the fake logger was not being initialized. nice catch